pkg/node/node.go

```diff
@@ -87,11 +87,23 @@ func (n *Node) AnnounceSignerPresence(
 		return nil, fmt.Errorf("failed to set broadcast channel filter: [%v]", err)
 	}
 
+	// TODO: Use generic type for representing address in node.go instead of
```

Comment:
Do we really need to convert the implementation to be chain agnostic? Isn't it too much effort and complication? I believe the only thing we wanted to achive was to log IDs of members that were not ready.

Reply:
That's why I left a TODO, I do not want to address it in this PR. Right now, `node.go` can support only Ethereum or - to be precise - chains that have the same algorithm for converting pub key to address. Even if we assume we do not want to look into the future and think about other host chains now, the interfaces are broken on the code level anyway: why to abstract `chain.Handle` if we can't support any chain? Why to abstract chain if we import `go-ethereum` from `node.go` and other code "above" `chain.Handle`. A similar problem is in `keep-core` but it's localized to `pkg/operator`, something we can replace in one week. Here, the effort will be much more significant and my goal was not to make it even more significant, that's all.